sky/clouds/service_catalog/gcp_catalog.py

```diff
@@ -392,11 +393,18 @@ def list_accelerators(
     new_infos = defaultdict(list)
     for info in acc_infos:
         assert pd.isna(info.instance_type) and pd.isna(info.memory), acc_infos
+        if info.accelerator_name.startswith('tpu'):
+            new_infos[info.accelerator_name].append(
+                info._replace(instance_type='TPU-VM',))
+            continue
         vm_types, _ = get_instance_type_for_accelerator(info.accelerator_name,
                                                         info.accelerator_count,
-                                                        region=region_filter)
+                                                        region=region_filter,
+                                                        use_spot=True)
+        #always has a vm type for GCP
```

Comment:
Could you elaborate on it a little bit? Like 
1. the acc name & count in `info` are retrieved from the table so we could definitely find a column in the original table
2. the format of the accelerator column in the GCP catalog is there are no values in the instance type column, reference here: https://github.com/skypilot-org/skypilot/blob/1e53317b5fcd6e1bd1b762afb23ae62e7ed72cd7/sky/clouds/service_catalog/data_fetchers/fetch_gcp.py#L327
3. So it will enable an specialized instance type https://github.com/skypilot-org/skypilot/blob/1e53317b5fcd6e1bd1b762afb23ae62e7ed72cd7/sky/clouds/service_catalog/gcp_catalog.py#L274-L284 or a default instance type https://github.com/skypilot-org/skypilot/blob/1e53317b5fcd6e1bd1b762afb23ae62e7ed72cd7/sky/clouds/service_catalog/gcp_catalog.py#L300 which is always existing in the table.